src/app/sh-http/sh-http.service.js

```diff
@@ -21,8 +21,7 @@ export default class ShHttpService {
     }
     put(url, body) {
         return this.http
-            .put(url, JSON.stringify(body), this._getOptions('Put', url))
-            .map(this._convertToJson);
+            .put(url, JSON.stringify(body), this._getOptions('Put', url));
```

Comment:
!fix
no need to send technical http information to the stream
use .error for failing requests